releases/release-1.31/release-notes/maps/pr-125442-map.yaml

```
@@ -0,0 +1,4 @@
+pr: 125442
+releasenote:
+  text: Graduate JobPodFailurePolicy to GA and lock
```

This PR also had `LockToDefault=true` set so maybe we can clarify:
```suggestion
  text: Graduated `JobPodFailurePolicy` to GA and locked it to it's default.
```